packages/@uppy/utils/src/Translator.js

```diff
@@ -50,12 +50,11 @@ module.exports = class Translator {
    * @license https://github.com/airbnb/polyglot.js/blob/master/LICENSE
    * taken from https://github.com/airbnb/polyglot.js/blob/master/lib/polyglot.js#L299
    *
-   * @param {string} phrase that needs interpolation, with placeholders
+   * @param {any} phrase that needs interpolation, with placeholders
```

Comment:
Hmm, I think we only allow string here and then all the jsx elements and such are inside the `options` object?

Reply:
All right, I noticed the implementation would take anything as there's nothing string specific.
```suggestion
   * @param {string} phrase that needs interpolation, with placeholders
```